pkg/k8shandler/cluster.go

```diff
@@ -17,6 +17,8 @@ import (
 var wrongConfig bool
 var nodes map[string][]NodeTypeInterface
 
+var aliasNeededMap map[string]bool
```

Comment:
These vars should really live as state on the elasticsearch object (maybe in status) as this construct as a global has the potential to grow without bounds.  I don't believe that will happen in our usage patterns but maybe something for tech debt.  
/lgtm

Reply:
i wonder if this is a problem other operators have had to solve and there is a best practice pattern for it. i'm a little wary of having it in status since it _could_ be modified externally